mkt/commonplace/tests/test_views.py

```diff
@@ -95,39 +99,6 @@ def test_region_included_in_fireplace_if_sim_info(self, mock_region):
             self.assertContains(res, 'data-region="testoland"')
 
 
-class TestAppcacheManifest(CommonplaceTestMixin):
```

Comment:
Either way, should probably put that in a different commit, it's a bunch of removed code that's completely orthogonal to the commit message.